index.Rmd

```diff
@@ -429,6 +435,144 @@ You can also browse the history of a repository from your Git hosting service, a
+
+```{block, type="alert alert-success"}
+> **Question 4**: what would be the location of this file: `././R/./.././R/./././function2.R`? How about `/./R/./.././R/./././function2.R`? What would be the meaning of `C:/Windows/cmd.exe` on Linux? Is it a relative or absolute file path?
+```
+
+<!--
+### Bigger data
+The `data/` directory of your project should indeed only contain relatively small data sets. When handling bigger remote sensing data sets, these should stay out of the project, where you store the rest of your data.
+### Example
+* Create 3 files in your `R/` directory (`ageCalculator.R`, `HelloWorld.R` and `minusRaster.R`) in which you will copy paste the respective functions.
+* Create a `main.R` script at the root of your project and add some code to it. The content of the `main.R` in that case could be something as below.
+```{r, eval=FALSE}
+# Name
+# Date
+library(raster)
+source('R/ageCalculator.R')
+source('R/HelloWorld.R')
+source('R/minusRaster.R')
+HelloWorld('john')
+ageCalculator(2009)
+# import dataset
+r <- raster(system.file("external/rlogo.grd", package="raster")) 
+r2 <- r 
+# Filling the rasterLayer with new values.
+r2[] <- (1:ncell(r2)) / 10
+# Performs the calculation
+r3 <- minusRaster(r, r2) 
+```
+### RStudio projects
+RStudio has a functionality called *projects* that allows organising your files a bit better. You may have learned that one of the first things to do when opening a R session is to set your working directory, using the `setwd()` command. When creating a new project, the working directory is automatically set to the root of the project, where your `main.R` is located. When working with RStudio projects you should not change the working directory. If you want to access files stored in your `data/` subdirectory, simply append `data/` to the beginning of the string leading to the file you want to load.
+**Note**: RStudio projects are specific to RStudio and are not usable with base R or with other R IDEs. The use of RStudio projects is optional and is merely for convenience. When using other IDEs you can assume the user will set the working directory to where the script is located.
+RStudio itself has integration with Git, and when creating a project there is an option to make it a git repository as well. However, in this lesson we will not be using this method, since it is specific to RStudio and does not work for Python or in other R IDEs. Git GUI, in contrast, is language-agnostic and standalone. So if you do create a project with RStudio, create it inside your cloned GitHub repository, and do **not** select that it creates a new git repository, then use Git GUI to handle all the changes you do in the repository. This will save you the confusion about how to handle it when we will come to Python lessons.
+-->
+
+# *RStudio* setup {#gettingstarted}
+For this course you can use RStudio to write and run scripts. You can access it from e.g. *Applications* → *Programming* → *RStudio* menu.
```

Comment:
Now there is no more "Programming" submenu, so it's more like "Applications" -> nine dots button -> "RStudio"